1.0/rules.rifps

```diff
@@ -0,0 +1,1686 @@
+Document(
+    Prefix(geo <http://www.opengis.net/ont/geosparql#>)
+    Prefix(geof <http://www.opengis.net/def/function/geosparql/>)
+
+    Group (
+        # geo:sfEquals
+        Forall ?f1 ?f2 ?g1 ?g2 ?g1Serial ?g2Serial (
+            ?f1[geo:sfEquals->?f2] :-
+            Or (
+                # feature – feature rule
+                And (
+                    ?f1[geo:hasDefaultGeometry->?g1]
+                    ?f2[geo:hasDefaultGeometry->?g2]
+                    ?g1[geo:gmlLiteral->?g1Serial]
```

Comment:
I think we need the property geo:asGML here instead of the datatype geo:gmlLiteral. So it would be ?g1[geo:asGML->?g1Serial]. The same thing goes for geo:asWKT and geo:wktLiteral.

We have a bit of a combinatorial explosion here because the functions like geof:sfEquals should work for mixed input - that is, comparing a WKT geometry with a GML geometry. We would need all these rules for (WKT, WKT) and (WKT, GML), (GML, WKT), (GML, GML). This becomes an even bigger problem when GeoJSON and KML are added. As an alternative, we could use the generic geo:hasSerialization property instead. geo:asWKT and geo:asGML are subPropertyOf geo:hasSerialization. This requires some RDFS reasoning support in the implementation though.

Reply:
It should be possible to create all permutations automatically. We could write a small script to do this and if more literal types are added just call the script with the newly available literal types.

Reply:
I wrote a script in #76 that would solve this issue, in case we would like to use RIF rules only and not rely on reasoning.

Reply:
Of course I produced the RIF rules doc in this PR  via a script too! I've added my scripts details to PR #76 and we will likely be able to produce a combined script for the profiles (it should be made available in both the 1.0 and 1.1 profiles with different inputs).

Once the script is finalised, I will ensure it is listed in the profiles correctly.

Reply:
Great! I will try to merge the script!